src/app/about-us/page.tsx

```diff
@@ -0,0 +1,29 @@
+import AboutUsPage from "@/components/about-us/AboutUsPage";
+import api from "../dataLayer/api";
+
+
```

Comment:
you should use generateMetadata on each page 
https://nextjs.org/docs/app/api-reference/functions/generate-metadata